packages/twenty-front/src/modules/settings/developers/components/SettingsWebhooksTable.tsx

```diff
@@ -24,13 +24,16 @@ export const SettingsWebhooksTable = () => {
     objectNameSingular: CoreObjectNameSingular.Webhook,
   });
 
+  const allWebhooksHaveEmptyTargetUrl =
```

Comment:
this does not look to be the right fix, we should still render the table if some webhooks have a targetUrl blank IMO

Reply:
The error is mainly coming from the `getUrlHostnameOrThrow(fieldItem.targetUrl)` function. You'll need to remove or modify this function to prevent this error.

Additionally, all webhook data fields are empty except for the `id` field, so I guess there is no chance to put anything inside the table.

NOTE: These empty URL values primarily occur when deleting webhooks, as the deletion of webhook function only clears the field values rather than removing the record from the database.

Reply:
Yeah, I forgot to explain what this variable does.

It just checks if the entire webhooks array has only empty targetUrl, then it is not going to render it, 

Otherwise, if there is, for example,e one empty webhook and one filled webhook, it is going to render the filled webhook and not going to render the empty one

@charlesBochet